kore/test/Test/Kore/Step/Simplification/Equals.hs

```diff
@@ -249,9 +252,10 @@ test_equalsSimplification_Or_Pattern =
                         { term = mkTop_
                         , predicate =
                             makeMultipleAndPredicate
-                                [ makeNotPredicate definedGWithSubstitution
-                                , makeNotPredicate definedF
+                                [ makeNotPredicate definedF
                                 , makeNotPredicate definedH
+                                , Simplification.Not.makePredicateNot
+                                    definedGWithSubstitution
                                 ]
```

Comment:
I changed the order of elements in this list so the test would pass. I don't know if that was important